report-jtreg-diff/src/main/java/io/jenkins/plugins/report/jtreg/main/diff/cmdline/JobsRecognition.java

```diff
@@ -211,7 +210,7 @@ public static boolean isNumber(String s) {
     }
 
     public static String getChangelogsNvr(File buildPath) {
-        return ConfigFinder.findInConfig(creteChangelogFile(buildPath), "nvr", "/build/nvr");
+        return new ConfigFinder(creteChangelogFile(buildPath), "nvr", "/build/nvr").findInConfig();
```

Comment:
if we move to display name, what we should, pls keep this method intact and add second method getBuildxmlDisplayName

Reply:
I created the getBuildxmlDisplayName() method, but it has no uses now.

Reply:
How come. it was use don many palces before, was it not? I mean the origianl call.